lib/rules/unit-blacklist/index.js

```diff
@@ -31,18 +40,64 @@ const rule = function(blacklistInput, options) {
         optional: true,
         actual: options,
         possible: {
-          ignoreProperties: validateObjectWithStringArrayProps
+          ignoreProperties: validateObjectWithStringArrayProps,
+          ignoreMediaFeatureNames: validateObjectWithStringArrayProps
         }
       }
     );
     if (!validOptions) {
       return;
     }
 
-    function check(node, value, getIndex) {
+    function checkMedia(node, value, getIndex) {
+      value = value.replace(/\*/g, ",");
```

Comment:
Let's leave this line out. Its sole purpose is handle parsing of calc functions - I don't think it's needed within media queries (and it's a bit of a hack so I'd like to not proliferate it if possible).

Reply:
Done in my last commit b6effdf